scripts/docker_push.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
-
 for tag in $*;do
-    echo "Tag ${IMAGE_NAME}:${DOCKER_TMP_TAG} as ${IMAGE_NAME}:${tag}"
-    docker tag "${IMAGE_NAME}:${DOCKER_TMP_TAG}" "${IMAGE_NAME}:${tag}"
-    echo "Push ${IMAGE_NAME}:${tag}"
-    docker push "${IMAGE_NAME}:${tag}"
+    for IMAGE in "$IMAGE_NAME_WORKER" "$IMAGE_NAME_WORKER";do
```

Comment:
Maybe invert the two loops so that all tags for an image are pushed before moving on to the next image.

Reply:
I have done that in cf45cf0. but I dont think it really matter because at that stage both images are built beforehand and the docker service of travis only pushed them.